src/main/java/frc/robot/utils/dynamicauto/ChoreoBuilder.py

```diff
@@ -0,0 +1,117 @@
+import json
+
```

Comment:
```suggestion

# File for constructing trajectories between each node (note and a shooting location).
# Locations each have an `allowed_destinations`, which states where it can possibly go to. Right now, this is set to every destination except itself.

```